src/modelmanager.cpp

```diff
@@ -828,6 +839,52 @@ void ModelManager::watcher(std::future<void> exitSignal) {
     SPDLOG_LOGGER_INFO(modelmanager_logger, "Stopped model manager thread");
 }
 
+void ModelManager::cleanerRoutine(uint32_t resourcesCleanupIntervalSec, uint32_t sequenceCleanerIntervalMinutes, std::future<void> cleanerExitSignal) {
+    SPDLOG_LOGGER_INFO(modelmanager_logger, "Started cleaner thread");
+
+    uint32_t resourcesCleanupIntervalMiliseconds = resourcesCleanupIntervalSec * 1000;
+    uint32_t sequenceCleanerIntervalMiliseconds = sequenceCleanerIntervalMinutes * 60 * 1000;
+
+    FunctorResourcesCleaner functorResourcesCleaner{*this};
+    FunctorSequenceCleaner functorSequenceCleaner{globalSequencesViewer};
+
+    ovms::cleanerRoutine(resourcesCleanupIntervalMiliseconds, functorResourcesCleaner, sequenceCleanerIntervalMiliseconds, functorSequenceCleaner, cleanerExitSignal);
```

Comment:
Why do we need 2 functions: `ovms::ModelManager::cleanerRoutine` and `ovms::cleanerRoutine`?

Reply:
This is for test purposes. We want to test cleaner functionality in the shortest possible time. Since sequence_interval is defined in minutes we decided to create function that takes inteval in miliseconds as an argument. Inside `ModelManager::cleanerRoutine` we create functors, which are also provided as arugments to the other function. This way it is easy to test whether cleanup is done in desired times.